### PR TITLE
[chores] Improvements to PR #220

### PR DESCRIPTION
**Pull request by nemesisdesign**
- improved docs
- reduced repetition (admin fields)
- other minor improvements

Related to #220

### CODE REVIEW

File: openwisp_radius/admin.py [hunk 1]
```diff
@@ -499,78 +499,61 @@ def has_delete_permission(self, request, obj=None):
...
+    pass
 
 
-class BirthDateField(forms.ChoiceField):
-    def prepare_value(self, value):
-        if not value:
-            value = OPTIONAL_SETTINGS.get('birth_date')
-        return super().prepare_value(value)
+class FallbackChoiceField(FallbackFieldMixin, forms.ChoiceField):
+    pass
```
Comment (nemesisdesign) [inline openwisp_radius/admin.py:518]: @NoumbissiValere DRY principle in action

File: openwisp_radius/tests/test_api.py [hunk 2]
```diff
@@ -1113,7 +1107,14 @@ def test_register_201(self):
...
     @mock.patch.object(
-        app_settings, 'OPTIONAL_REGISTRATION_FIELDS', optional_settings_params
+        app_settings,
+        'OPTIONAL_REGISTRATION_FIELDS',
+        {
+            'first_name': 'disabled',
+            'last_name': 'allowed',
+            'birth_date': 'disabled',
+            'location': 'mandatory',
+        },
```
Comment (nemesisdesign) [inline openwisp_radius/tests/test_api.py:1117]: @NoumbissiValere unless the attribute `optional_settings_params` would be reused in other tests, there's no reason to store it as a class attribute, better pass it directly here. It makes the test declaration faster to read.

Comment (NoumbissiValere) [reply openwisp_radius/tests/test_api.py:1117]: Ok @nemesisdesign

File: openwisp_radius/base/models.py [hunk 3]
```diff
@@ -149,8 +149,8 @@
     'allowed to register via the user registration API.'
 )
 _GET_OPTIONAL_FIELDS_HELP_TEXT = _(
-    'Chose if this user field should be disabled, allowed or mandatory '
-    'during user registration.'
+    'Whether this field should be disabled, allowed or mandatory '
+    'in the user registration API.'
```
Comment (NoumbissiValere) [inline openwisp_radius/base/models.py:153]: This changes the help text on the model. so we need to run `./tests/manage.py makemigrations` for the migration to be updated.

**Review by NoumbissiValere — COMMENTED** [review]
@nemesisdesign , Thank you for these corrections.

File: openwisp_radius/base/models.py (hunk 3 above)
Comment (nemesisdesign) [reply openwisp_radius/base/models.py:153]: updated

**Pull request merged**